test/interval_tests/complex.jl

```diff
@@ -14,7 +14,7 @@ end
 
 @testset "Complex functions" begin
```

Comment:
OK, I see your point about `±` - I'll change it as you suggest to use `@round_up` and `@round_down`.